skype/skype-ps/skype/New-CsOnlinePSTNGateway.md

```diff
@@ -101,22 +116,42 @@ Allows some codecs to be excluded when media is being negotiated between Media P
 Type: String
 Parameter Sets: (All)
 Aliases:
-Applicable: Skype for Business Online
+Applicable: Microsoft Teams
 Required: False
 Position: Named
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -FailoverResponseCodes
+If Direct Routing receives any 4xx or 6xx SIP error code in response to an outgoing Invite the call is considered completed by default. (Outgoing in this context is a call
+from a Teams client to the PSTN with traffic flow: Teams Client -> Direct Routing -> SBC -> Telephony network). Setting the SIP codes in this parameter forces Direct Routing
+on receiving the specified codes try another SBC (if another SBC exists in the voice routing policy of the user). Find more information in the "Reference" section of "Phone
+System Direct Routing" documentation.
+
+Setting this parameter overwrites the default values, so if you want to include the default values, please add them to string string.
```

Comment:
is "please add them to the string" maybe?

Reply:
Fixed